crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -296,8 +297,12 @@ def proposal?(file)
         raw_data.key?("attributes") && raw_data.key?("deployment")
       end
 
-      def migrate_database(time)
-        Crowbar::Migrate.migrate!
+      def migrate_database(time, migration_level = nil)
+        if migration_level
+          Crowbar::Migrate.migrate_to(migration_level)
```

Comment:
What happens if that migration level is not known yet? (ie: use new backup on old code)